imports/ui/components/GoogleMapComponent.js

```diff
@@ -120,6 +131,10 @@ class GoogleMapComponent extends Component {
         {this.props.isMarkerShown && (
           <Marker position={this.props.currentLocation} />
         )}
+        <StaticModal show={this.state.showAddAudio} body={<AddAudioForm/>}
+                onHide={this.handleAddAudio.bind(this)}
```

Comment:
`.bind` creates a new function with passed context. Instead bind in constructor, or define `handleAddAudio` as a field instead of method. Fields are instance variables and are bound to `this`.

```js
class A {
  // method
  handleAddAudio () {
  }

  // field
  handleAddAudioNew = () => {
  }
}